content/blog/esc-db-secrets-rotation-launch/index.md

```diff
@@ -0,0 +1,193 @@
+
+## Introducing ESC Rotated Secrets for Databases
+
+Pulumi ESC’s [Rotated Secrets](/docs/esc/integrations/rotated-secrets/) capability now extends to your critical databases, bringing automation and enhanced security to credential lifecycle management. We're launching support for:
+
+*   [**PostgreSQL**](/docs/esc/integrations/rotated-secrets/postgres/) (Including instances hosted on AWS RDS/Aurora or in your Private VPCs)
+*   [**MySQL**](/docs/esc/integrations/rotated-secrets/mysql/) (Including instances hosted on AWS RDS/Aurora or in your Private VPCs)
+*   [**Snowflake**](/docs/esc/integrations/rotated-secrets/snowflake-user/)
+
+Here’s how ESC Rotated Secrets for Databases tackles the challenges:
+
+*   **Automated Rotation Schedules:** Effortlessly define rotation frequencies (daily, weekly, monthly) that align with your security policies, eliminating manual toil and ensuring consistency.
+*   **On-Demand Rotation:** Instantly rotate credentials in response to security events, personnel changes, or policy updates with a single click or command.
+*   **Seamless ESC Environment Integration:** Manage rotated database credentials alongside all your other application configurations within ESC Environments, leveraging composition and environment inheritance.
+*   **Two-Secret Strategy:** ESC maintains both the current and previous valid credentials simultaneously. This prevents application downtime during rotation, as instances can gracefully transition to the new credential.
+*   **Auditing and Tracking:** Maintain a complete audit trail of credential rotations, including who initiated the rotation and when, simplifying compliance and governance.
+*   **Admin and Creator Control:** Securely configure rotation using privileged database "managing user" credentials stored within ESC, keeping them separate from the rotated credentials consumed by applications.
+*   **Configure Webhooks for Automation:** Trigger notifications, CI/CD pipelines (e.g., Pulumi Deployments to update application stacks), or custom workflows upon successful rotation, keeping dependent systems synchronized.
+
+{{% notes type="info" %}}
+The following setup guide provides a detailed walkthrough for [**PostgreSQL**](/docs/esc/integrations/rotated-secrets/postgres/) hosted within an AWS private VPC, which requires a Lambda-based connector. Please refer to the [**MySQL**](/docs/esc/integrations/rotated-secrets/mysql/) or [**Snowflake**](/docs/esc/integrations/rotated-secrets/snowflake-user/) documentation page for instructions on these two databases.
+{{% /notes %}}
+
+## How to Set Up and Use PostgreSQL Rotation (AWS VPC Example)
+
+Setting up rotation for PostgreSQL databases running inside an AWS VPC involves [creating the required users](/docs/esc/environments/rotation/db-preparation/) within your database, deploying the rotator Lambda infrastructure via a Pulumi template, configuring ESC environments, and scheduling the rotation.
```

Comment:
Should we detail our approach here? We are defining "rotator Lambda infra" without first defining it.